playbook.tex

```diff
@@ -8,6 +8,14 @@
 \usepackage{lipsum}
 \usepackage[sfdefault]{roboto}
 \usepackage[T1]{fontenc}
+\usepackage{soul}												%to underline links
+\usepackage[hidelinks, urlcolor=blue, colorlinks=true]{hyperref}				%to create nice looking links (blue)
+\hypersetup{
+bookmarks=true,												%to create bookmarks in the PDF
```

Comment:
Don't move comments to the right. It might look nice on your screen, but people with smaller screens will have no idea what the heck is going on. Also, the comment for `bookmarks=true` should explain what exactly those bookmarks are. The current comment is pretty much implied by the code itsself.